_translator-files/po/es/Tips-Tricks-More.po

```diff
@@ -350,57 +352,61 @@ msgid ""
 "     | `s` | Solo  |\n"
 "     | `m` | Mute  |\n"
 msgstr ""
+"     | letra de control | Control Jamulus |\n"
+"     |--------:|---------|\n"
+"     | `f` | Fader |\n"
+"     | `p` | Paneo   |\n"
```

Comment:
No reason for the extra trailing spaces, is there?

Reply:
Fixed.